### ISSUE TITLE
Source formatting settings

### ISSUE THREAD

**Issue by vanowm** [loganch/AutoIt-VSCode#130]
@loganch do you think, maybe you could share your formatting settings, so they could be used for PRs?

**Comment by loganch** [comment]
My formatting is based on the airbnb-base through eslint and prettier, which depends on the `eslint`, `eslint-config-airbnb-base`, `eslint-config-prettier`, `eslint-plugin-import`, `eslint-plugin-prettier` and `prettier` devDependencies. I also have the eslint extension https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint installed that surfaces the formatting issues into the problem tab.

**Comment by vanowm** [comment]
Maybe eslint settings should be included in the repo?

**Comment by loganch** [comment]
They already were ([.eslintrc.json](https://github.com/loganch/AutoIt-VSCode/blob/master/.eslintrc.json), [.prettierrc.json](https://github.com/loganch/AutoIt-VSCode/blob/master/.prettierrc.json)). 

I just realized that I didn't include a link to the [Prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode), which I also have installed, but I feel like it's only for formatting and wouldn't have an effect on seeing issues everywhere since all of that is covered through ESLint.

**Comment by vanowm** [comment]
That is strange, I have these extensions installed and get a lot of errors:
![image](https://user-images.githubusercontent.com/511517/235301359-83fee424-f21f-4e34-b05a-885bcc5b2d4e.png)

**Comment by loganch** [comment]
That actually looks about right for ai_command.js, here's how it looks on my side:

![image](https://user-images.githubusercontent.com/7883107/235312524-8222e8a4-e356-47bc-a057-9ce71ab001ec.png)

Are you seeing any problems show for say, extension.js or ai_signature.js

**Comment by vanowm** [comment]
No, these are fine, however, ai_commands.js is pretty much unusable:
![image](https://github.com/loganch/AutoIt-VSCode/assets/511517/25bfdfb6-6b5a-4614-9f31-95acbad1e64f)
